frontend/src/components/Admin/adminExportImport.tsx

```diff
@@ -49,89 +49,250 @@ export const AdminExportConnect = () => {
           {message}
         </p>
       )}
+    </div>
+  );
+};
 
-      <div className="bg-gray-50 border border-gray-200 rounded-xl p-6 text-sm text-gray-700">
-        <h3 className="font-semibold text-gray-800 mb-2 text-lg">
-          📄 Exemple de fichier CSV :
-        </h3>
-        <pre className="bg-white p-4 rounded-lg border text-xs overflow-x-auto">
-          {`name,description,location,start_at,end_at,capacity,is_open
-Permanence 1,Accueil matin,A001,2025-05-01T08:00,2025-05-01T10:00,10,false
-Permanence 2,Accueil après-midi,A002,2025-05-02T14:00,2025-05-02T16:00,15,false`}
-        </pre>
-        <p className="mt-4 text-xs text-gray-500">
-          Le fichier doit être encodé en UTF-8 et utiliser une virgule comme séparateur. Les dates doivent être au format
-          <code className="ml-1 font-mono bg-gray-200 px-1 rounded">
-            aaaa-mm-jjThh:mm
-          </code>.
-        </p>
+
+export const AdminImportPlannings = () => {
+  const [planning, setPlanning] = useState<File | null>(null);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+  const [message, setMessage] = useState<string>("");
+  const [selectedPlanning, setSelectedPlanning] = useState<string>("");
+
+  const handleFileChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setError(null);
+    setMessage("");
+    if (e.target.files && e.target.files.length > 0) {
+      const selected = e.target.files[0];
+      if (selected.type !== "application/pdf") {
+        setError("Seuls les fichiers PDF sont autorisés");
+        setPlanning(null);
+      } else {
+        setPlanning(selected);
+      }
+    }
+  };
+
+  const handleImport = async (planningName: string) => {
+    if (!planning) {
+      setError("Veuillez sélectionner un fichier PDF avant d’importer.");
+      return;
+    }
+
+    setLoading(true);
+    setError(null);
+    setMessage("");
+    setSelectedPlanning(planningName);
+
+    try {
+      const formData = new FormData();
+      formData.append("planningFile", planning);
+
+      const response = await importPlannings(formData);
+      setMessage(response.message || `Importation réussie pour ${planningName} !`);
+    } catch (err) {
+      console.error("Erreur lors de l’importation du planning", err);
+      setError("Une erreur est survenue pendant l’importation.");
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  return (
+    <div className="max-w-xl mx-auto p-8 bg-white rounded-2xl shadow-xl">
+      <h2 className="text-3xl font-bold text-center text-gray-900 mb-6">
+        Importer les plannings au format PDF
+      </h2>
+
+      {/* Rappel des règles de nommage */}
+      <p className="text-center text-sm text-red-600 font-medium mb-4">
+        ⚠️ Le fichier doit être nommé en minuscules, sans accents, au format <br />
+        <code className="font-mono">filière.pdf</code> (ex: <code>tc.pdf</code>, <code>bachelor.pdf</code>)
```

Comment:
sans l'accent me parrait plus logique du coup
```suggestion
        <code className="font-mono">filiere.pdf</code> (ex: <code>tc.pdf</code>, <code>bachelor.pdf</code>)
```